python_sdk/infrahub_sdk/ctl/utils.py

```diff
@@ -24,6 +35,69 @@ def init_logging(debug: bool = False) -> None:
     logging.getLogger("infrahubctl")
 
 
+def catch_exception(
+    which_exception: Union[type[Exception], Iterable[type[Exception]]] = Exception,
+    console: Console = None,
```

Comment:
This feels like something that mypy would complain about, might be that the rules are slightly incorrect.

Reply:
![image](https://github.com/opsmill/infrahub/assets/736209/119f567e-83ad-41d9-bdd0-94c7828a249b)

Reply:
:)